python/pyspark/pandas/namespace.py

```diff
@@ -2554,7 +2553,7 @@ def resolve_func(psdf, this_column_labels, that_column_labels):
         if isinstance(obj, Series):
             num_series += 1
             series_names.add(obj.name)
-            new_objs.append(obj.to_frame(DEFAULT_SERIES_NAME))
+            new_objs.append(obj.to_frame())
```

Comment:
Just a question, after this change, do we support old Pandas versions still?

Reply:
Yes, actually this followups a bug fix from Pandas(https://github.com/pandas-dev/pandas/issues/15047) so I believe it should be fine.

Reply:
Thank you for the confirmation, @itholic .